tests/packages/extra/test_extra_group_pkgs.py

```diff
@@ -0,0 +1,12 @@
+from lib.commands import SSHCommandFailed
+from pkgfixtures import host_with_saved_yum_state
+
+def test_extra_group_packages_url_resolved(host, extra_pkgs):
+    for p in extra_pkgs:
+        host.ssh(['yumdownloader', '--resolve', '--urls', p])
+        
+def test_extra_group_packages_can_be_installed(host_with_saved_yum_state, extra_pkgs):
+    # Just try to install all packages together. Installing them one by one
+    # takes too much time due to the generation of the initrd.
+    # If yum fails then it will be caught by pytest.
```

Comment:
This is a tacit contract with pytest, so we don't really need to mention it. It is true for almost anything we do in our tests.

```suggestion
```

Reply:
makes sense :+1: